akka-stream/src/main/boilerplate/akka/stream/javadsl/SourceCreate.scala.template

```diff
@@ -9,13 +9,34 @@ import akka.stream.scaladsl.JavaConverters._
 
 trait SourceCreate {
 
+  /**
+   * Creates a `Source` by using a `FlowGraph.Builder[Unit]` on a block that expects
+   * a [[FlowGraph.Builder]] and returns an [[Outlet]].
+   */
```

Comment:
Nice boy-scouting! Although I am the original offender, may I ask you to wave your magic wand also on the other templates? Thanks a bunch!